File: tests/bench/python/prime_sieve.py
def sieve(n):
    limit = n
    count = 0
    for i in range(2, limit):
        is_prime = True
        j = 2
        while j * j <= i:
            if i % j == 0:
                is_prime = False
                break
            j += 1
        if is_prime:
            count += 1
    return count

print(sieve(5000))

File: tests/bench/js/prime_sieve.js
function sieve(n) {
    let count = 0;
    for (let i = 2; i < n; i++) {
        let isPrime = true;
        for (let j = 2; j * j <= i; j++) {
            if (i % j === 0) { isPrime = false; break; }
        }
        if (isPrime) count++;
    }
    return count;
}
console.log(sieve(5000));
